src/main/java/Duke.java

```diff
@@ -1,10 +1,67 @@
+import java.util.Scanner;
+import java.util.*;
```

Comment:
You should only import packages that you need and not import everything from java.util